osquery/CMakeLists.txt

```diff
@@ -29,13 +29,18 @@ else()
 endif()
 
 if(WIN32)
+  set(WINDOWS_DEP_DIR "$ENV{ChocolateyInstall}/lib")
```

Comment:
Can we move this search path into the `ADD_OSQUERY_LINK_INTERNAL` CMake macro that's edited above?